samples/bluetooth/llpm/src/main.c

```diff
@@ -457,7 +466,25 @@ void main(void)
 		return;
 	}
 
-	advertise_and_scan();
+	while (true) {
+		printk("Choose device role - type m (master role) or s (slave role): ");
+
+		char input_char = console_getchar();
+
+		printk("\n");
+
+		if (input_char == 'm') {
+			printk("Master role. Starting scanning\n");
+			scan_start();
```

Comment:
The `scan_init` (Line.450) can be moved to master role only.

Reply:
Fixed